CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/glyphs/GlyphManagerOpenGLController.java

```diff
@@ -75,11 +75,6 @@ public void bind(GL3 gl, int coordinatesUniformLocation, int coordinatesTextureU
         bindGlyphs(gl, glyphsUniformLocation, glyphsTexureUnit);
     }
 
-    public void dispose(GL3 gl) {
```

Comment:
Was this method removed because it wasn't used?

Reply:
Yep, this method is called by deleteTextures which is never used.

Reply:
This has clearly not been implemented, but I'd be cautious here as dispose is a core function in GL so I suspect that it should be implemented to avoid memory leaks on the graphics card.

Reply:
@formalhaut69 is looks like this is being used so can you please restore this method?

https://github.com/constellation-app/constellation/blob/247c2e69563f30f9cede92ca10dc0610f2557359/CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/utilities/SharedDrawable.java#L233.

Reply:
I think that this could be removed based on my comment https://github.com/constellation-app/constellation/pull/446/files#r413149947